assets/src/scss/pages/_page.scss

```diff
@@ -121,3 +121,16 @@
     outline: 0 none;
   }
 }
+
+.bottom-navigation {
+  height: 70px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-weight: 700;
+  font-family: var(--font-family-heading);
+
+  a:hover {
+    text-decoration: none;
```

Comment:
Is that a request from the design team? Seems weird to me that we want no hover effects, especially after the accessibility work we've done 🤔